txpool/txpool.go

```diff
@@ -59,6 +59,7 @@ var (
 	ErrTipAboveFeeCap          = errors.New("max priority fee per gas higher than max fee per gas")
 	ErrTipVeryHigh             = errors.New("max priority fee per gas higher than 2^256-1")
 	ErrFeeCapVeryHigh          = errors.New("max fee per gas higher than 2^256-1")
+	ErrNonceExists             = errors.New("transaction with nonce already exists in pool")
```

Comment:
I think this error is not something `ethereum` has standardized and in their code. I know that some of the users (I think Nexon was the one), wanted these errors to be one on one with `ethereum`, and we even had a PR for that.

I know we don't have transaction replacement in `txpool` (like `ethereum`), but I am not sure if we want this issue done like this. Let's wait for product to decide.